integration/apps/nekbone/nekbone.py

```diff
@@ -54,25 +54,32 @@ def __init__(self, add_barriers=False):
         benchmark_dir = os.path.dirname(os.path.abspath(__file__))
         self._nekbone_path = os.path.join(benchmark_dir, 'nekbone/test/example1/')
         self._num_rank_per_node = 2
+        self._input_name = 'data.rea'
 
     def get_rank_per_node(self):
         # TODO: use self._machine_file to determine
         return self._num_rank_per_node
 
-    def get_bash_setup_commands(self):
+    def trial_setup(self, run_id, output_dir):
         size = 10000  # this size varies per system
-        input_file = textwrap.dedent('''
+        input_data = textwrap.dedent('''
```

Comment:
The data.rea should always be the same for every trial.

Doesn't this belong in experiment_setup?

Reply:
We got rid of experiment_setup.  The issue is that there are N app_conf objects, but only one start to the experiment.  Instead, the implementer of the trial_setup() is responsible for checking for file existence.